cdap-unit-test/src/test/java/io/cdap/cdap/spark/SparkTest.java

```diff
@@ -196,7 +192,7 @@ public void testDynamicSpark() throws Exception {
     }
   }
 
-  @Test
+//  @Test
```

Comment:
Is this disabled intentionally?

Reply:
Yes, it was a flaky test that is why I disabled for testing in local environment. I must have missed this. Let me enable it now.